src/components/study-container.jsx

```diff
@@ -537,7 +535,7 @@ export function StudyContainer({ view, onChangeTab }) {
                         });
 
                     // Check if the current root network has changed and if there's a current selected node
-                    if (previousCurrentRootNetwork !== currentRootNetworkUuid && currentNode) {
+                    if (currentNode) {
```

Comment:
this is an error.
We must override the currentNode when rootNetwork change because even if the tree is the same currentNode could have different build status in a root network or another.

Reply:
I keep the condition as before but using previous value in ref.. but i think it not really necessary because in the useEffect we have already a condition to trigger the call of loadTree